blueprint/api.apib

```diff
@@ -1227,6 +1229,64 @@ This endpoint will return one of two possible responses, depending on if the slu
 
     + Attributes (Slugged Route Organization Response)
 
+# Group Stripe Accounts
+
+This resource represents a Stripe account. It can be retrieved to see properties on the account like its current e-mail address or if the account is enabled yet to make live charges.
```

Comment:
I would also specify the relationship between an account and an organization, how that works.